scripts/dev/dockerfile_generator.py

```diff
@@ -1,9 +1,11 @@
 import jinja2
 import argparse
 import os
+import sys
+from typing import List, Dict, Union
 
 
-def operator_params(files_to_add):
+def operator_params(files_to_add: List[str]) -> Dict[str, Union[bool, str, List[str]]]:
```

Comment:
if Union[bool, str, List[str]] is used a lot it may be worth defining it (the hardest part would be to find a relevant name ;) )

Reply:
`Dict[str, Union[bool, str, List[str]]]` is even less readable than `Dict` I think! Was this a mypy error?